### PR TITLE
Fix wrong byte units

### PR DESCRIPTION
**Pull request by cih9088**
### Summary
- Fixed byte units ([wiki](https://en.wikipedia.org/wiki/Byte#Multiple-byte_units))

### Checklist

- [x] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added e2e tests for validation.
- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

File: otx/cli/tools/train.py [hunk 1]
```diff
@@ -131,7 +131,7 @@ def get_args():
         required=False,
         help="Size of memory pool for caching decoded data to load data faster. "
         "For example, you can use digits for bytes size (e.g. 1024) or a string with size units "
-        "(e.g. 7KB = 7 * 2^10, 3MB = 3 * 2^20, and 2GB = 2 * 2^30).",
+        "(e.g. 7KiB = 7 * 2^10, 3MB = 3 * 10^6, and 2G = 2 * 2^30).",
```
Comment (eunwoosh) [inline otx/cli/tools/train.py:134]:
```suggestion
        "(e.g. 7KiB = 7 * 2^10, 3MB = 3 * 10^6, and 2GiB = 2 * 2^30).",
```

Comment (cih9088) [reply otx/cli/tools/train.py:134]: `G`-ish unit is supported as well.

Comment (cih9088) [reply otx/cli/tools/train.py:134]: It was on purpose as `G`-ish unit is supported as well.

Comment (eunwoosh) [reply otx/cli/tools/train.py:134]: Ah I see.

**Review by eunwoosh — DISMISSED** [review]

**Review by vinnamkim — APPROVED** [review]
LGTM.

**Review by vinnamkim — CHANGES_REQUESTED** [review]
Would you fix this as well?
https://github.com/openvinotoolkit/training_extensions/blob/3299c66a211b03170896d994937b62ef52caa081/docs/source/guide/get_started/quick_start_guide/cli_commands.rst?plain=1#L203

**Comment by cih9088** [comment]
> Would you fix this as well?

Fixed in https://github.com/openvinotoolkit/training_extensions/pull/1998/commits/958e5ed72523d8643349f1be9b71c527365ec65e

**Review by sungmanc — APPROVED** [review]

**Review by vinnamkim — APPROVED** [review]
LGTM.

**Pull request merged**